src/Components/Settings/Settings.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import style from './Settings.module.scss'
+import { SettingsContext } from '../../Context/SettingsContext'
+
+const Settings = () => {
+    const [ settings, setSettings ] = useContext(SettingsContext)
+    
+    const onChange = (event) => {
+        setSettings({ 
+            ...settings, 
+            [event.target.id]: +event.target.value
+        })
+    }
+
+    return (
+        <div className={style.settingsContainer}>
+            <h1 className={style.settingsHeader}>Settings</h1>
+            <p>Currently, you are set to work for {settings.workInterval} minutes and then take a {settings.breakInterval} minute break</p>
+            <div className={style.intervalContainer}>
+                <p className={style.intervalLabel}>Select your work interval:</p>
+                {<input 
+                    type='number' 
+                    min={0} 
+                    max={240} 
+                    id='workInterval' 
+                    placeholder={settings.workInterval} 
+                    onChange={onChange}
+                    className={style.minutesInput}
+                />}
```

Comment:
May I ask Y this lil input is in `{}`?

Reply:
I think it's leftover from when I commented out the original `select`!! Just noticed that if you comment out something in a block of JSX it automatically wraps it in curly brackets.